tests/test_views.py

```diff
@@ -3208,6 +3208,29 @@ def setUpClass(cls):
         cls._original_enable_email_subscriptions = settings.ENABLE_EMAIL_SUBSCRIPTIONS
         settings.ENABLE_EMAIL_SUBSCRIPTIONS = True
 
+    def test_user_cancel_and_choose_help_mailing_list(self):
+        user = AuthUserFactory()
```

Comment:
Split this test into two tests with descriptive function names please. Try to keep unittests as small as possible